src/mjolnir/ingest_transit.cc

```diff
@@ -263,6 +263,7 @@ void setup_stops(Transit& tile,
     node->set_traversability(static_cast<uint32_t>(Traversability::kBoth));
   }
   node->set_name(tile_stop.stop_name);
+  // TODO(nils): this is usually empty and should only override an agency's timezone
```

Comment:
this will lead to problems: an agency's timezone is commonly the source of truth. need to fix that too

Reply:
remove entirely in favor of our own stuff

Reply:
default to agency, then override with stops.txt, then fallback to own timezone db

Reply:
this is too much work to be worth it. we default to our own timezone anyways which really shouldn't disagree with an agency's timezone.. if it does, likely the operator messed up and should've overriden this stop's timezone